### DISCUSSION TITLE
Did not receive any github sponsorship withdrawals.

### DISCUSSION THREAD

**Discussion by glepnir** [community/community#55356] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I've received some sponsorships recently. But they didn't go to my bank card. I also don't have any button to use the withdrawal. so how do i withdraw sponsorship money to my bank card

**Comment by DHayes10** [comment]
Hi @glepnir, 

Thank you for reaching out. I have reviewed your account and issued a payout which should arrive in your account 2023-05-22. 

Best, 
GitHub Sponsors

**Comment by glepnir** [comment]
So the minimum amount is $20? I also received some sponsorships a few months ago, but they didn't reach my bank card
